src/Files.Uwp/UserControls/StatusBarControl.xaml

```diff
@@ -24,8 +24,7 @@
         Height="32"
         Padding="8,0"
         HorizontalAlignment="Stretch"
-        VerticalAlignment="Stretch"
-        ColumnSpacing="12">
```

Comment:
This fixes a small issue where in the Status bar when no items are selected there's a 24px space (instead of 12px) between element count and size.

Reply:
Can this be done in a seperate pull request for tracking purposes?

Reply:
Sure, sorry 👍